kill-react-scripts.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash
+#
+# Kills all existing react-scripts processes and thus frees default port 3000
+# to be used by new react-scripts process.
 
-# kills any existing react-scripts processes and thus frees
-# default port 3000 to be used by new react-scripts instance
+echo "Killing all existing react-scripts processes ..."
 
-echo "killing any existing react-scripts instances"
+PROCESSES=`ps -ef | grep -i -E "react-scripts.*(start|test)" | grep -v grep`
```

Comment:
I renamed a few variables because `processes` and `process` are so close (and thus confusing).